### ISSUE TITLE
[Bug Report] goTo vertical scroll does not work on iOS

### ISSUE THREAD

**Issue by raybog** [vuetifyjs/vuetify#6914]
### Versions and Environment
**Vuetify:** 1.5.9
**Vue:** 2.6.10
**Browsers:** Safari
**OS:** Mac OS 10.14.3

### Steps to reproduce
On iOs device (any browser) or Safari (mac os) click any of the menus (one, two, three).

### Expected Behavior
It should scroll, after menu click, vertically to the corresponding IDs

### Actual Behavior
It does not scroll on, menu click, on iOs device (any browser) or Safari (mac os). 

After scrolling the window down a bit then it works fine.

### Reproduction Link
<a href="https://ww9nq0kq48.codesandbox.io" target="_blank">https://ww9nq0kq48.codesandbox.io</a>

### Other comments
Found problem in goTo function and sending trough pull request soon.

**Comment by madebycaliper** [comment]
@raybog I've been trying to debug a similar issue. It appears the goTo functionality only breaks if you're at the *very top* of the iOS safari window. If you move even 1px down, the clicks begin to work.

I tested by opening the XCode Simulator and entering `window.scrollTo(0,1)` clicking my buttons (and you're codesandbox link) and the scroll works.  I also then confirmed there was no scroll after `window.scrollTo(0,0)` and clicking again. Any updates on the progress of the fix? (I read some of #6915 ). Any help would be great.

**Comment by haliloncen** [comment]
I'm having the same problem. Is there a solution?

**Comment by madebycaliper** [comment]
@haliloncen I tried merging some of the code from that PR into my local copy of Vuetify but it didn't seem resolve the issue (might have done it wrong). Because it was time-sensitive for me I just added a reliable package I've worked with before, [vue-scrollto](https://github.com/rigor789/vue-scrollto), which is very minimal in terms of config/markup.

It's a shame to have to add more code to my vendor.js file but it's working!
